hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/MultiDCAwareService.java

```diff
@@ -40,6 +57,7 @@ public MultiDCOffsetChangeSummary moveOffset(Topic topic, String subscriptionNam
 
         if (!dryRun) {
             adminTool.retransmit(new SubscriptionName(subscriptionName, topic.getName()));
+            clusters.forEach(clusters -> waitUntilOffsetsAreMoved(topic, subscriptionName));
```

Comment:
Why this should be invoked under `!dryRun` ? 🤔

Reply:
Because in `!dryRun` we trigger consumers to change offsets. Therefore we have to wait until offsets are moved. In dryRun we don't move offsets (only read them), so we don't have to wait  for this operation  to complete.